datafusion/sqllogictest/test_files/string/string_view.slt

```diff
@@ -935,12 +935,19 @@ logical_plan
 01)Projection: to_timestamp(test.column1_utf8view, Utf8("a,b,c,d")) AS c
 02)--TableScan: test projection=[column1_utf8view]
 
-## Ensure no casts for nullif
+## Ensure no casts for NVL
 query TT
+EXPLAIN SELECT NVL(column1_utf8view, 'a') as c2 FROM test;
+----
+logical_plan
+01)Projection: nvl(test.column1_utf8view, Utf8View("a")) AS c2
+02)--TableScan: test projection=[column1_utf8view]
+
+## Ensure no casts for nullif
+query error DataFusion error: SQL error: ParserError\("Expected: an SQL statement, found: logical_plan"\)
```

Comment:
I recheck this and found it is error, why is that?

Reply:
Looks like a merge error, fixing.

Reply:
That was probably my bad -- sorry